doc/formats/morphology.rst

```diff
@@ -0,0 +1,142 @@
+.. _formatsmorph:
+
+Morphology
+----------
+
+Arbor supports morphologies described using the SWC file format and the NeuroML file format.
+
+SWC
+~~~
+
+Arbor supports reading morphologies described using the
+`SWC <http://www.neuronland.org/NLMorphologyConverter/MorphologyFormats/SWC/Spec.html>`_ file format.
+
+SWC files may contain comments, which are stored as metadata. And a blank line anywhere in the file is
+interpreted as end of data.
+
+The description of the morphology is encoded as a list of samples with an id,
+an `x,y,z` location in space, a radius, a tag and a parent id. Arbor parses these samples, performs some checks,
```

Comment:
Please add 'separated by whitespace'. At least I keep forgetting.

Reply:
Sure. BTW, this whole file was copied with no changes. I'll add a separate note for other reviewers.